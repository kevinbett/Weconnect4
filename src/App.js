import React from 'react';
import Routes from "./router";

class App extends React.Component {
  render() {
    return (
      <div>
        <Routes/>
      </div>
      
    );
  }
}

export default App;
