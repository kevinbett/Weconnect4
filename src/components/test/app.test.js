import App from "../../App"
import { shallow } from "enzyme"

describe('App component',() => { 
    it ('renders app component', () => { 
        expect(shallow(<App/>)).toMatchSnapshot()
    });
} )