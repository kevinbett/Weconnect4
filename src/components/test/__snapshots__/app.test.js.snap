// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders app component 1`] = `
<div>
  <routes />
</div>
`;
